src/components/Home/Aditya/Week3/utils/utils.ts

```diff
@@ -0,0 +1,14 @@
+export const removeWhitespace = (str: string): string => {
+  let currentTask = str.replace(/\s+/g, ' ');
+  if (currentTask) {
```

Comment:
if currentTask is always going to be a string do we really need this check? Try to refactor this code.